pyproject.toml

```diff
@@ -327,7 +327,6 @@ module = [
     "neptune.internal.background_job",
     "neptune.internal.container_structure",
     "neptune.internal.credentials",
-    "neptune.internal.disk_queue",
```

Comment:
Old files are removed from toml, but new files aren't added?

Reply:
This is a list of files to be excluded from type checking by `mypy`. This effectively means that from now we have to have proper typing across new files.